wgsl/index.bs

```diff
@@ -1066,11 +1062,15 @@ image_storage_type
 variable_decoration_list
   : ATTR_LEFT (variable_decoration COMMA)* variable_decoration ATTR_RIGHT
 
+literal_or_ident
+  : FLOAT_LITERAL
+  | INT_LITERAL
+  | UINT_LITERAL
+  | STRING_LITERAL
```

Comment:
I don't think we want STRING_LITERAL. This is really only for a few cases like the imports. IDENT would cover it I think.